spec/mail/message.mdown

```diff
@@ -276,12 +273,12 @@ The properties of the Message object submitted for creation MUST conform to the
 - **bcc**:  Optional. Overrides a "Bcc" in the *headers*.
 - **replyTo**: Optional. Overrides a "Reply-To" in the *headers*.
 - **subject**: Optional. Defaults to the empty string (`""`).
-- **date**: Optional. If included, the server SHOULD wait until this time to send the message (once moved to the outbox mailbox). Until it is sent, the send may be cancelled by moving the message back out of the outbox mailbox. If the date is in the past, the message must be sent immediately. A client may find out if the server supports delayed sending by querying the server's *capabilities* object (see section 1).
+- **date**: Optional. Overrides a "Date" in the *headers*.
```

Comment:
1. *headers*'s keys are supposed to be lowercased, so this should probably say "date".
2. What does "overrides" mean here? I'm reading it as (assuming that the server stores mails in RFC5322 format) the server is supposed to replace the "date" header's value with whatever is supplied here (after converting it from RFC 3339 to RFC 5322 format) before storing it on disk.  Is the intention here to be able to supply the original headers, but fix up the date without having to parse 5322-style dates?

Reply:
1. Yep, I'll lower case these.
2. The date/to/cc/bcc/from/replyTo fields are all parsed representations that could also be represented in raw form in the headers property. This is simply specifying what the server should do if the client supplies both for some reason: Take the parsed one and blat the one in the "headers" property. I'll rewrite this description to be clearer.